src/services/YotiSessionCompletionProcessor.ts

```diff
@@ -56,6 +56,17 @@ export class YotiSessionCompletionProcessor {
   	this.generateVerifiableCredential = GenerateVerifiableCredential.getInstance(this.logger);
   }
 
+  isTaskDone(data: any, taskType: string): boolean {
```

Comment:
Is there any way to be more specific with this type?

Reply:
Will action in next PR